index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="header_container">
+        <nav class="navbar">
+            <h2>Welcome</h2>
+            <img src="/assets/Union.png" alt="bg">
+        </nav>
+        <section class="intro_section">
+            <h2> Hey there. I'm Favour <br>
+                <span>I'm a software developer</span></h2>
+            <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        </section>
+        <div class="social-icon">
+            <a href="www.facebook.com"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+            <a href="www.twitter.com"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            <a href="www.github.com/fayob"><i class="fa fa-github" aria-hidden="true"></i></a>
+            <a href="www.linkedin.com/in/abimbola-ade" class="fa fa-linkedIn"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+        </div>
+    </header>
+  </body>
+</html>
```

Comment:
Hey, your application is not matching the Figma template of your choosing. Please follow the design you choose to its core. Your app should have the exact same layout as your chosen design which includes background, margins, paddings, font size, font family, colors, etc.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/185333265-17e50c29-3923-437f-a703-d4f51594fe4f.png) | ![image](https://user-images.githubusercontent.com/33268927/185333161-539d5230-ad04-409d-b78f-3acffcc035ab.png) |